common/src/config/game-support/ddr.ts

```diff
@@ -0,0 +1,186 @@
+import {FmtNum, FmtPercent, FmtScoreNoCommas} from "../../utils/util";
+import {ClassValue, zodNonNegativeInt} from "../config-utils";
+import {p} from "prudence";
+import {z} from "zod";
+import type {INTERNAL_GAME_CONFIG, INTERNAL_GAME_PT_CONFIG} from "../../types/internals";
+import {IIDXDans} from "./iidx";
```

Comment:
code isn't formatted or lint checked; run `pnpm lint-fix` inside the server folder to fix this stuff.

(you'll have `pnpm` installed and set up correctly inside the docker container)